pkgs/applications/video/mpv/default.nix

```diff
@@ -19,6 +19,11 @@
   , libcdio          ? null
   , libcdio-paranoia ? null
 
+, vulkanSupport ? true
```

Comment:
```suggestion
, vulkanSupport ? stdenv.isLinux
```
Let's not brake darwin build.

Reply:
I was not aware this was gonna break the darwin build. Fixed :)